src/component/project/ProjectAccess/ProjectAccessTable/ProjectRoleCell/ProjectRoleCell.styles.tsx

```diff
@@ -2,6 +2,8 @@ import { makeStyles } from 'tss-react/mui';
 
 export const useStyles = makeStyles()(theme => ({
     cell: {
-        padding: theme.spacing(1, 1.5),
+        padding: theme.spacing(0, 1.5),
```

Comment:
Huh, cool trick, did not know about https://mui.com/material-ui/customization/spacing/#multiple-arity 👍